Update README.md

mariano-resendiz opened the pull request:
Left 3 Answers and a few questions.

prof-lankham commented on week-01/README.md:
```diff
@@ -1,5 +1,17 @@
 ## Course Structure Quiz Problems
 
+
+[Course Structure Quiz, Problem 1]
```
Please update so that contributed content appears alongside related content, rather than above the Instructions portion.

mariano-resendiz committed:
Update README.md

prof-lankham reviewed (changes requested):
Title incorrect

prof-lankham reviewed (changes requested):
Questions needed for all Problems/Recipes and should be relative to their content; also, questions should not be repeated

mariano-resendiz committed:
Update README.md

mariano-resendiz committed:
Week 1 Quiz: Round 2.5

Fixed some questions and answers to reduce redundancy and to be more appropriate to context.

mariano-resendiz committed:
Update README.md

mariano-resendiz committed:
Update README.md

mariano-resendiz committed:
Rename README.md to Quiz-1.md

mariano-resendiz committed:
Update Quiz-1.md
